model-import/cli-import.mdx

```diff
@@ -1,300 +1,172 @@
+
+### Runtime
+
+Runtime in Inferless refers to the environment and configuration in which your model runs. It includes:
+
+1. System packages
+2. Python packages
+3. Custom shell commands
+
+### Create and deploy with runtime
+
+1. Create a new file called `inferless_runtime_config.yaml` with below data
+    
+    ```bash
+    build:
+  cuda_version: "12.1.1"
   python_packages:
-  - huggingface-hub==0.11.0
-  - transformers==4.36.1==6.0.1
-  - diffusers==0.24.0
-```
-
-
-If you don't have the requirements in the same repo you can build the config.yaml using the below documentation 
-
-[https://docs.inferless.com/model-import/bring-custom-packages](https://docs.inferless.com/model-import/bring-custom-packages)
-
-### Push the runtime 
-
-```
-inferless runtime upload
-```
-
-![](/images/cli9.png)
-
-The CLI will ask you to update the config automatically, or else you can update manually in inferless.yaml
-
-### Using an existing runtime : 
-
-```
-inferless runtime select --id 
-```
-
-![](/images/cli10.png)
-
-
-### Creating a Volume 
-
-```
-inferless volume create
-```
-
-### List all the volumes 
-
-Use this command to get the id of the volume
-
-```
-inferless volume list 
-```
-
-### Using an existing volume 
-
-```
-inferless volume select --id 
-```
-
-### Copy data from machine to  Volume
-
-copy a file 
-
-```
-inferless volume cp -s /path/to/local/file -d infer://region-1/<volume-name>/<folder>/file  
-```
-
-copy the entire folder 
-
-```
-inferless volume cp -r -s /path/to/local/folder -d infer://region-1/<volume-name>/<folder> 
-
-```
-
-### List the data in Volume
-
-```
-inferless volume ls -i <volume-id> -p /path 
-```
-
-
-### Copy data Volume to local machine
-
-```
-inferless volume cp -s infer://region-1/<volume-name>/<folder> -d /path/to/local/file 
-```
-
-### Delete the data in the Volume 
-
-```
-inferless volume rm -p infer://region-1/<volume-name>/<folder> 
-```
-
-
-
-
-
-
-**Depreciated - Input / Output Json** 
-
-
-* `input.json`This file will have the key for the input parameter. Whenever you change the name of the key in the `app.py`, update it accordingly.
-
-* `output.json`This file will have the `name` of the output key that the `def infer` the function is going to return.
-
-Update the input.json / Output Json as per your model. 
-
-```
-// Input.json 
-{
-  "inputs": [
-    {
-      "data": [
-        "Image of horse near beach"
-      ],
-      "name": "prompt",
-      "shape": [
-        1
-      ],
-      "datatype": "BYTES"
-    }
-  ]
-}
-```
-
-```
-// Output.json 
-
+    - "accelerate==0.33.0"
+    - "torch==2.4.0"
+    - "transformers==4.44.0"
+    - "diffusers==0.30.0"
+    ```
+
+2. Run the below command to create the runtime
+    
+    ```bash
+    inferless runtime create --name <runtime_name> --path ./inferless_runtime_config.yaml
+    ```
+
+3. Deploy with runtime
+    
+    ```bash
+    inferless deploy --gpu t4 --region <regionname> --runtime <runtimename> 
```

Comment:
runtime_name instead of runtimename , either add region in above command and remove from both command

Reply:
inferless deploy --gpu t4 --region <region_name> --runtime <runtime_name>